move/squid/sources/squid/versioned/squid_v0.move

```diff
@@ -0,0 +1,161 @@
+module squid::squid_v0;
+
+use sui::clock::Clock;
+
+use version_control::version_control::{Self, VersionControl};
+
+use axelar_gateway::channel::{Self, Channel, ApprovedMessage};
+
+use its::its::ITS;
+use its::service;
+use squid::coin_bag::{Self, CoinBag};
+use squid::swap_info::{Self, SwapInfo};
+
+public struct SquidV0 has store {
+    channel: Channel,
+    coin_bag: CoinBag,
+    version_control: VersionControl,
+}
+
+// -----------------
+// Package Functions
+// -----------------
+public(package) fun new(version_control: VersionControl, ctx: &mut TxContext): SquidV0 {
+    SquidV0 {
+        channel: channel::new(ctx),
+        coin_bag: coin_bag::new(ctx),
+        version_control,
+    }
+}
+
+public(package) fun channel(self: &SquidV0): &Channel {
+    &self.channel
+}
+
+public(package) fun version_control(self: &SquidV0): &VersionControl {
+    &self.version_control
+}
+
+public(package) fun coin_bag_mut(self: &mut SquidV0): &mut CoinBag {
+    &mut self.coin_bag
+}
+
+public fun start_swap<T>(
```

Comment:
move public functions to the top, and add section separators

Reply:
should not have been public, fixed